.gitignore

```diff
@@ -26,3 +26,6 @@ yarn.lock
 .idea/
 out/
 .idea_modules/
+
+# TypeScript
+**/*.js
```

Comment:
Please add a new line at EOF.
I'll give you a reference written in Korean.
Let's make it our convention too.
ref: https://velog.io/@doondoony/posix-eol